backend-models/service-tests/accessedservice-tests.txt

```diff
@@ -0,0 +1,129 @@
+
+accessedservice.php
+
+-------------------
+
+********
+* pre-req
+********
+
+(checklogin() is TRUE && (hasAccess($userid, $cid, 'w') || isSuperUser($userid,) is TRUE))
```

Comment:
To make sure added text arent confused with code i write it with only caps and each statement on a different row.
as such.
(checklogin() IS TRUE 
&& 
(hasAccess($userid, $cid, 'w') || isSuperUser($userid,) )) IS TRUE